src/components/Expensicons/Icons.js

```diff
@@ -0,0 +1,39 @@
+import ICON_NAMES from './ICON_NAMES';
+import ClipboardIcon from '../../../assets/images/clipboard.svg';
+import CopyLinkIcon from '../../../assets/images/link-copy.svg';
+import ExpensifyCashLogoIcon from '../../../assets/images/expensify-cash.svg';
+import MailIcon from '../../../assets/images/mail.svg';
+import PencilIcon from '../../../assets/images/pencil.svg';
+import PinIcon from '../../../assets/images/pin.svg';
+import TrashIcon from '../../../assets/images/trashcan.svg';
+
+export default {
+    [ICON_NAMES.CLIPBOARD]: {
+        icon: ClipboardIcon,
+        isAssetColored: false,
```

Comment:
Maybe it would be good to add some context for what `isAssetColored` refers to? First time I'm seeing this and I'm not too sure how it will be used.